projects/ion/src/lib/message/message.component.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { CommonModule } from '@angular/common';
+import { RenderResult, render, screen } from '@testing-library/angular';
+
+import { IonMessageComponent } from './message.component';
+import { IonMessageProps, MessageStatusType } from './types';
+import { IonIconComponent } from '../icon';
+
+const defaultValue: IonMessageProps = {
+  label: 'Message',
+};
+
+export const icontypes: MessageStatusType[] = [
+  'custom',
+  'positive',
+  'negative_alert',
+  'negative_error',
+  'warning',
+  'info',
+];
+
+const sut = async (
+  customProps: IonMessageProps = defaultValue
+): Promise<{
+  fixture: RenderResult<IonMessageComponent>;
+  element: HTMLElement;
+}> => {
+  const fixture = await render(IonMessageComponent, {
+    componentProperties: customProps,
+    imports: [CommonModule, IonIconComponent],
+  });
+  const element = fixture.fixture.nativeElement as HTMLElement;
+  return {
+    fixture,
+    element,
+  };
+};
+
+describe('MessageComponent', () => {
+  it('should render the positive type as default', async () => {
+    const { fixture, element } = await sut({
+      ...defaultValue,
+      type: 'positive',
+    });
+    fixture.detectChanges();
+    expect(element.getAttribute('data-type')).toBe('positive');
+  });
+
+  it('should have a message', async () => {
+    await sut();
+    expect(screen.getAllByText(defaultValue.label)).toHaveLength(1);
+  });
+});
+
+describe('', () => {
+  it.each(icontypes)('should render %s type', async type => {
+    const { fixture, element } = await sut({
+      ...defaultValue,
+      type: type,
+    });
+    fixture.detectChanges();
+    expect(element.getAttribute('data-type')).toBe(`${type}`);
+  });
```

Comment:
```suggestion
  it.each(icontypes)('should render %s type', async type => {
    const { fixture, element } = await sut({
      ...defaultValue,
      type: type,
    });
    fixture.detectChanges();
    expect(element.getAttribute('data-type')).toBe(`${type}`);
  });
```
I don't think you really need do create another describe here. If you really need it, I suggest that you give it a name, like 'message type'.